src/renderer/lib/validate.ts

```diff
@@ -1,28 +1,15 @@
-export const isValidIpAddress = (ip: string): boolean => {
-  if (ip === "localhost") {
-    return true;
-  }
-
-  const chunks = ip.split(".");
-  if (chunks.length !== 4) {
-    return false;
-  }
+import { isIPv4, isIPv6 } from "is-ip";
```

Comment:
```suggestion
import { isIPv4, isIPv6 } from "net";
```
What's the difference from just importing these from the `net` package?

Reply:
we can't use `net.isIP` from renderer. I could add IPC but that seems overkill for this.

Reply:
Oh right. We could consider exposing it over the context bridge, similar to how to expose the the electron-log functions. E.g.
```
     // inside of preload.ts
     log: log.functions,
     net: { isIPv4, isIPv6},
```
and then in the actual usage of it:
```
    const { isIPv4, isIPv6 } = window.electron.net; 
```

Although it does reduce a dependency, it also makes it harder to test logic and components since it now requires them to be injected over context bridge.

I'll defer it to you, but I'm happy to approve this change once the tests/build is passing.